docs/pages/asking_for_input.rst

```diff
@@ -165,6 +165,7 @@ Creating a custom style could be done like this:
 .. code:: python
 
     from prompt_toolkit.shortcuts import prompt
+    from prompt_toolkit.styles import Style
```

Comment:
Thanks for this! Would it be possible to merge the import with the next line?

Reply:
Yup, updated commit.

Reply:
Thanks for this!